admin.php

```diff
@@ -52,7 +52,7 @@ public function handle()
         $cmd = $INPUT->extract('cmd')->str('cmd');
         switch ($cmd) {
             case 'export':
-                $exportfile = $conf['tmpdir'] . '/' . $this->db->getDBName() . '.sql';
+                $exportfile = $conf['tmpdir'] . '/' . $this->db->getDbName() . '.sql';
```

Comment:
Hmm. Is the tmp dir publically available? Shouldnt it be protected by data's .htaccess?